### DISCUSSION TITLE
Atari ST info

### DISCUSSION THREAD

**Discussion by sideburn** [BlueSCSI/BlueSCSI-v2#104] [Help]
Is there a wiki or any documentation on steps required to setup BlueSCSI on an Atari ST? 

Assuming for starters an ACSI to SCSI adapter is needed. After that not sure about drivers. Would you use the same procedure as you would setting up an UltraSatan?

**Comment by mynameistroy** [comment]
Yes it would be the exact same as it will just look like an ASCI device to the Atari so you can use ICD/PPera/ HDDRIVER and there shouldn't be any issues.
